test/checkoutput/sample_input.md

```diff
@@ -0,0 +1,80 @@
+# Example
+
+This directory includes example logger setup allowing users to easily check and test impact of logging configuration. 
+
+Below we can see examples of how some features work.
+
+## Default
+
+Run:
+```console
+go run ./staging/src/k8s.io/component-base/logs/example/cmd/logger.go
+```
+
+Expected output:
+```
+I0605 22:03:07.224293 3228948 logger.go:58] Log using Infof, key: value
+I0605 22:03:07.224378 3228948 logger.go:59] "Log using InfoS" key="value"
+E0605 22:03:07.224393 3228948 logger.go:61] Log using Errorf, err: fail
+E0605 22:03:07.224402 3228948 logger.go:62] "Log using ErrorS" err="fail"
+I0605 22:03:07.224407 3228948 logger.go:64] Log message has been redacted. Log argument #0 contains: [secret-key]
+```
+##Simple example
+
+Run:
+```console
+echo hello
+```
+
+Expected output:
+```
+hello
+```
+
+## JSON 
+
+Run:
+```console
+go run /home/prasadc/dev/k8s/repo/k8s/k1/src/k8s.io/kubernetes/staging/src/k8s.io/component-base/logs/example/cmd/logger.go --logging-format json
```

Comment:
I still see the `/home/prasadc/dev/...` path here.

As this is now inside `test/checkoutput`, you probably need to use `../../staging/k8s.io/kubernetes/staging/src/k8s.io/component-base/logs/example/cmd/logger.go`.

However, when I try `go run .` inside this directory, the tool gets stuck:
```
test/checkoutput$ go run .
v log level is  1 args is  [.]
Processing root directory  .
Successful Main package Lookup: path name is  k8s.io/kubernetes/test/checkoutput
Parsing mdfile  README.md  in dir  ./.
^Csignal: interrupt
```

Running it under a debugger I see that it invokes `go run .` as the command whose output it captures, not the one specified here. Therefore it gets stuck in an endless recursion.

Logging the commands that it invokes would be very useful here.

Reply:
Fixed .